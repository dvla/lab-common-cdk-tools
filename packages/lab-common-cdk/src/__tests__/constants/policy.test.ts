import { labconst } from '../..';

describe('Tests Policy statements', () => {
    test('Tests services', () => {
        expect(labconst.policy.COMPREHEND_ALLOW.toStatementJson()).toMatchSnapshot();
        expect(labconst.policy.REKOGNITION_ALLOW.toStatementJson()).toMatchSnapshot();
        expect(labconst.policy.TEXTRACT_ALLOW.toStatementJson()).toMatchSnapshot();
    });
});
