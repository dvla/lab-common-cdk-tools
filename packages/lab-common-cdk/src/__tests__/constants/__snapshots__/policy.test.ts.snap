// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests Policy statements Tests services 1`] = `
Object {
  "Action": "comprehend:*",
  "Effect": "Allow",
  "Resource": "*",
}
`;

exports[`Tests Policy statements Tests services 2`] = `
Object {
  "Action": "rekognition:*",
  "Effect": "Allow",
  "Resource": "*",
}
`;

exports[`Tests Policy statements Tests services 3`] = `
Object {
  "Action": "textract:*",
  "Effect": "Allow",
  "Resource": "*",
}
`;
